src/material/tabs/_tabs-common.scss

```diff
@@ -343,6 +343,14 @@ $mat-tab-animation-duration: 500ms !default;
   flex-grow: 1;
   overflow: hidden;
   z-index: 1;
+  border-bottom-style: solid;
+  @include token-utils.use-tokens(
+      tokens-mat-tab-header.$prefix,
+      tokens-mat-tab-header.get-token-slots()
+  ) {
+    @include token-utils.create-token-slot(border-bottom-width, divider-height);
+    @include token-utils.create-token-slot(border-bottom-color, divider-color);
```

Comment:
There's an option to position the header at the bottom which we may have to account for: https://material.angular.io/components/tabs/examples#tab-group-header-below

Reply:
ooh good catch, PR lgtm otherwise

Reply:
Nice! Added this to the dev-app, I didn't realize this was an option. Thanks for catching this